container/features/src/main/resources/features.xml

```diff
@@ -690,6 +690,15 @@
       <bundle>mvn:org.opennms/opennms-icmp-api/${project.version}</bundle>
     </feature>
 
+    <feature name="opennms-icmp-best" description="OpenNMS :: ICMP :: Best Match" version="${project.version}">
```

Comment:
Is this feature automatically loaded in the Minion container? (I couldn't find any reference)